cleverhans/loss.py

```diff
@@ -24,7 +26,30 @@ def __init__(self, model, hparams=None, attack=None):
     :param attack: callable, the attack function for adv. training.
```

Comment:
shouldn't this NOT be "callable" in the docstring anymore?

Reply:
Thanks, updated it to specify an Attack